src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -92,7 +92,7 @@ static View()
         /// Creates a new instance of a view.
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
-        public View() : this(Interop.View.New(), true)
+        public View() : this(Interop.View.New(), true, null)
```

Comment:
This change may occure some strange issues.

(Maybe current codes are not works like this but..) Some subclass may can call `base(cPtr, cMemoryOwn, shown)` constructor directly. in that case, that subclass cannot use default Theme.

Instead of change this line, I hope to change  `internal View(global::System.IntPtr cPtr, bool cMemoryOwn, bool shown = true)`.
Make that function call `this(cPtr, cMemoryOwn, null, shown)`, and move whole jobs into `internal View(global::System.IntPtr cPtr, bool cMemoryOwn, ViewStyle viewStyle, bool shown = true) : base(cPtr, cMemoryOwn)`

Reply:
done